src/typeChecker/__tests__/source1Typed.test.ts

```diff
@@ -655,7 +655,7 @@ describe('binary operations', () => {
       const x11: string = x3 + x2; // error, boolean + string, return type string
       const x12: number = x1 + x4; // no error, number + any, return type number
       const x13: string = x5 + x2; // no error, any + string, return type string
-      const x14: string = x1 + x2; // error, number + string, return type string
+      const x14: number = x1 + x2; // error, number + string, return type number (follows left side)
```

Comment:
Should the other lines that follow right side also have `(follows right side)` for consistency?

Reply:
The extra info is added because this one line is the special case where one side is `number` and the other side is `string`, in which case the left type will be used. The other cases are more straightforward (follow the side where the type is valid, else return `any`), so the extra info is not necessary.